api/src/index.js

```diff
@@ -2,29 +2,36 @@ import "dotenv/config";
 import express from "express";
 import cors from "cors";
 import bodyParser from "body-parser";
-import knex from "./database_client.js";
 import nestedRouter from "./routers/nested.js";
+import firstMeal from "./routers/firstMeal.js";
+import allMeals from "./routers/allMeals.js";
+import futureMeals from "./routers/futureMeals.js";
+import lastMeal from "./routers/lastMeal.js";
+import pastMeals from "./routers/pastMeals.js";
 
 const app = express();
 app.use(cors());
 app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
-});
-
-// This nested router example can also be replaced with your own sub-router
 apiRouter.use("/nested", nestedRouter);
+apiRouter.use("/first-meal", firstMeal);
+apiRouter.use("/all-meals", allMeals);
+apiRouter.use("/future-meals", futureMeals);
+apiRouter.use("/future-meals", pastMeals);
 
 app.use("/api", apiRouter);
+app.use("/first-meal", firstMeal);
+app.use("/all-meals", allMeals);
+app.use("/future-meals", futureMeals);
+app.use("/last-meal", lastMeal);
+app.use("/past-meals", pastMeals);
```

Comment:
in "real life" I would probably suggest you to use one routing group here for meals. This way you would be able to achieve a folder structure like this: 
```
\routers
  \meals
    index.js // --here your routes will go and all get/post/etc requests with referece to functions defines in the rest of the files
   getAll.js
   getFirst.js
   getFuture.js
   ...
   ```
   Sometimes it's also nice to include type of queries in the file name so you know how to easily find stuff :)
   
   This is just a suggestion, but usually `.use` is for routing groups and then particular requests have either separate files or one collective file. 
   
   Here is a quite nice example: https://www.freecodecamp.org/news/rest-api-design-best-practices-build-a-rest-api/

Reply:
I couldnt be sure about if we are going to use  .../api/first-meal or  .../first-meal  thats why I kept them both, but since its API I guess we would use .../api/first-meal. So I updated the code :)